src/commands/get-recipient-user-referral-sheet-upload-url.ts

```diff
@@ -0,0 +1,23 @@
+import { Command } from '@boostercloud/framework-core'
+import { Register } from '@boostercloud/framework-types'
+import { UserRegistered } from '../common/roles'
+import { UploadFileService } from '../services/upload-file-service'
+
+@Command({
+  authorize: [UserRegistered],
+})
+export class GetRecipientUserReferralSheetUploadUrl {
+  public constructor(readonly recipientUserId: string) {}
+
+  public static async handle(command: GetRecipientUserReferralSheetUploadUrl, register: Register): Promise<string> {
+    return UploadFileService.getUrlToUploadFile(
+      GetRecipientUserReferralSheetUploadUrl.getReferralSheetPath(command.recipientUserId)
+    )
+  }
+
+  private static getReferralSheetPath(recipientUserId: string): string {
+    const currentDate = new Date().toISOString().slice(0, 10)
```

Comment:
I think it would be better to use a date formatter here

Reply:
I don't know what do you mean, something like this?

`const currentDate  = new Date().toLocaleDateString("es-ES")); `

Reply:
The problem that I see here is that using the `slice(0,10)` we are just getting the date. Could be possible to try to upload several documents the same day?